upstream/composer.json

```
@@ -13,19 +13,18 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
```

It might be better to leave this line alone.  We don't actually want to enforce 8.3 as a minimum PHP version until D11.
```suggestion
        "php": ">=8.1",
```